sycl/include/sycl/ext/oneapi/matrix/matrix-jit.hpp

```diff
@@ -191,6 +191,22 @@ joint_matrix_mad(Group sg, joint_matrix<T1, M, K, LayoutA, Group> &mA,
                       PI_INVALID_DEVICE);
 #endif // __SYCL_DEVICE_ONLY__
 }
+
+template <typename Group, typename T, size_t NumRows, size_t NumCols,
+          matrix_layout Layout>
+inline __SYCL_ALWAYS_INLINE void
+joint_matrix_fill(Group sg,
+                  joint_matrix<T, NumRows, NumCols, Layout, Group> &res,
+                  const T v) {
+  (void)sg;
```

Comment:
nit: might be worth to put a comment here, why we kept (sg) parameter (to be aligned with other AMX API?, to may be replaced __spirv_CompositeConstruct with some other instruction in case if we are not satisficed without breaking API/ABI with returning 'sg' back?)